### PR TITLE
chore(litmus-portal): Adding redux setup for the frontend

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: Soumya Ghosh Dastidar <gdsoumya@gmail.com>

**What this PR does / why we need it**:

Fixed Redux Setup

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: litmus-portal/frontend/src/config/index.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default {
   environment: process.env.NODE_ENV,
-  api: {
-    url: process.env.REACT_APP_API_HOST || 'http://localhost:8080/api/v1',
+  analytics: {
+    url: 'https://staging-hub.litmuschaos.io/api/community',
```
Comment (rajdas98) [inline litmus-portal/frontend/src/config/index.ts:4]: put this in a .env file under two section prod and staging

File: litmus-portal/frontend/src/components/VerifyCommit/index.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ function VerifyCommit() {
                   <ButtonOutline
                     isDisabled={false}
                     handleClick={() => {
-                      console.log('Edit');
+                      // console.log('Edit');
```
Comment (rajdas98) [inline litmus-portal/frontend/src/components/VerifyCommit/index.tsx:97]: remove the code comments in all the places

File: litmus-portal/frontend/src/config/index.ts (hunk 1 above)
Comment (gdsoumya) [reply litmus-portal/frontend/src/config/index.ts:4]: I can add this to the k8s-manifest as env?

File: litmus-portal/frontend/src/components/VerifyCommit/index.tsx (hunk 2 above)
Comment (gdsoumya) [reply litmus-portal/frontend/src/components/VerifyCommit/index.tsx:97]: This is not my code, it was giving eslint warnings so I commented it.

File: litmus-portal/frontend/src/config/index.ts (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/frontend/src/config/index.ts:4]: Not to the k8s manifest. In frontend dir, put .env file for all the environment variable which will be used by the frontend.
you can call it by `process.env.ENV_VAR`

File: litmus-portal/frontend/src/components/VerifyCommit/index.tsx (hunk 2 above)
Comment (rajdas98) [reply litmus-portal/frontend/src/components/VerifyCommit/index.tsx:97]: Is it possible to remove it?

Comment (gdsoumya) [reply litmus-portal/frontend/src/components/VerifyCommit/index.tsx:97]: Not sure, they might have kept it for some reason.

File: litmus-portal/frontend/src/config/index.ts (hunk 1 above)
Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/config/index.ts:4]: I think we should keep the .env files for API secrets and config file is good for keeping pubic APIs...this setup was used by alden...and I liked the approach.

Comment (rajdas98) [reply litmus-portal/frontend/src/config/index.ts:4]: Okay, for now we can take this directly. But in future we have to create .env and after chaos hub code merged to the master we will get the prod url.

**Pull request merged**